src/js/shield_defs.js

```diff
@@ -2098,15 +2098,29 @@ export function loadShields() {
       bottom: 6,
     },
   };
-  ["Clark", "Washoe"].forEach(
-    (county) =>
-      (shields[`US:NV:${county}`] = pentagonUpShield(
-        3,
-        15,
-        Color.shields.blue,
-        Color.shields.yellow
-      ))
-  );
+
+  // County shields (only Washoe in this case)
+  ["Washoe"].forEach((county) => {
+    shields[`US:NV:${county}`] = pentagonUpShield(
+      3,
+      15,
+      Color.shields.blue,
+      Color.shields.yellow
+    );
+  });
+
+  // Clark County Route 215 gets its own special highway shield and is not the same as a normal county route
+  shields["US:NV:Clark"] = {
+    spriteBlank: "shield_us_nv_clark",
+    textColor: Color.shields.blue, // #003882
+    textLayout: textConstraint("ellipse"),
+    padding: {
+      left: 2.5,
+      right: 2.5,
+      top: 5,
+      bottom: 7,
```

Comment:
The blue number collides into the blue pentagon, making it difficult to read. This shrinks the number to about the same size as the number on the state route shield, but still readable:

```suggestion
      left: 6,
      right: 6,
      top: 6,
      bottom: 8,
```